### ISSUE TITLE
Issue with Step 6 of Demo

### ISSUE THREAD

**Issue by AlexWillcox** [DeepLabCut/DeepLabCut#87]
I'm trying to work through the demo using an IPython terminal through Anaconda Prompt on a windows 7 PC, however when I type the command 'run ../../../train.py' I get the following error:

![error](https://user-images.githubusercontent.com/44472853/47509095-44140e00-d86d-11e8-9182-aeef81a63b3e.png)

Please could you suggest what I am doing wrong- I may be going about the whole thing incorrectly, as this is the first time I have tried using python or tried training a neural network.

Thanks!

**Comment by AlexEMG** [comment]
I think, you should restart the ipython/python session and then do it again (the weights are already loaded in the tf session and this is why you cannot load it again). This can also be done by: ```tf.reset_default_graph()```

**Comment by AlexWillcox** [comment]
Thank you for your replies, I restarted the session and using the command "run ../../../train.py" from the directory DeepLabCut/pose-tensorflow/models/reachingJan30-trainset95shuffle1/train and this time, the script appeared to run. I did, however get a DataLossError (picture posted).

Is this the loss described in step 6? I so, should I continue running the script multiple times?

![error2](https://user-images.githubusercontent.com/44472853/47553040-f3002a80-d8fd-11e8-8e2f-60a4d7e94e14.png)

**Comment by AlexEMG** [comment]
are you training from scratch, or from a particular snapshot? [also what version of TF?]

**Comment by AlexWillcox** [comment]
I'm training using the demo files, so using the mouse reaching video from Mathis et al., 2017 and this is the first time I've tried working through the demo. Also, I downloaded everything including anaconda and TensorFlow this week, so I believe I'm using TensorFlow 1.11, but will have to double-check that when I'm back in the office.

**Comment by MMathisLab** [comment]
Also, no that error is not what you get when training normally. From some older closed issues, this is what it should look like: 

```mackenzie@c997c82acb00:~/DeepLabCut-master/pose-tensorflow/models/reachingJan30-trainset95shuffle1/train$ TF_CUDNN_USE_AUTOTUNE=0 C
UDA_VISIBLE_DEVICES=0 python3 ../../../train.py
I tensorflow/stream_executor/dso_loader.cc:135] successfully opened CUDA library libcublas.so.8.0 locally
I tensorflow/stream_executor/dso_loader.cc:135] successfully opened CUDA library libcudnn.so.5 locally
I tensorflow/stream_executor/dso_loader.cc:135] successfully opened CUDA library libcufft.so.8.0 locally
I tensorflow/stream_executor/dso_loader.cc:135] successfully opened CUDA library libcuda.so.1 locally
I tensorflow/stream_executor/dso_loader.cc:135] successfully opened CUDA library libcurand.so.8.0 locally
.....

I tensorflow/core/common_runtime/gpu/gpu_device.cc:885] Found device 0 with properties:
name: GeForce GTX 1080
major: 6 minor: 1 memoryClockRate (GHz) 1.835
pciBusID 0000:0c:00.0
Total memory: 7.92GiB
Free memory: 7.80GiB
I tensorflow/core/common_runtime/gpu/gpu_device.cc:906] DMA: 0
I tensorflow/core/common_runtime/gpu/gpu_device.cc:916] 0: Y
I tensorflow/core/common_runtime/gpu/gpu_device.cc:975] Creating TensorFlow device (/gpu:0) -> (device: 0, name: GeForce GTX 1080,
pci bus id: 0000:0c:00.0)
iteration: 0 loss: 0.0002 lr: 0.005 
```

@AlexEMG was asking if you are running this step for the first time, i.e. a new network or you are loading from a snapshot. If not, you need to be sure to run this: ``tf.reset_default_graph()`` as it seems you have a session running. 

At this point it's not clear CUDA 9.2 and TensorFlow 1.11 are supported, please see the installation guide for our recommendations on which TF versions we support: https://github.com/AlexEMG/DeepLabCut/blob/master/docs/installation.md

**Comment by AlexWillcox** [comment]
Hi,

Sorry it's taken me so long to get back to you, but I uninstalled tensorflow 1.11 and replaced it with tensorflow 1.1 but am getting the same error- even though I also ran "tf.reset_default_graph() ".

Please could you suggest where I might be going wrong?

**Comment by MMathisLab** [comment]
Can you tell us about your system? Windows, which graphic card, etc? Please check the Cuda/driver/TF are compatable - see Installation guide. Maybe consider joining the slack for help too... see README page.
